source/cfp.rst

```diff
@@ -0,0 +1,63 @@
+Call For Proposals
+==================
+
+We are planning to use `papercall.io`_ for handling CFP proposals.
+
+Daniele has set up a form with the following questions:
+
+Talk details
+------------
+
+Title
+  No help text
+Abstract
+  The abstract is the 300 character elevator pitch for this talk.
+Talk Format
+  What format is this talk best suited for?
+  Options: Talk; Workshop or tutorial; Code clinic, lunchtime event or other themed gathering; Poster
+Audience Level
+  Who is the best target audience for this talk?
+  Options: All; Beginner; Intermediate; Advanced
+Description
+  The description is the public abstract of your talk. The description will be seen by reviewers during the CFP process and will eventually be seen by the attendees of the event.
+  You should make the description of your talk as compelling and exciting as possible. Remember, you're selling both the organizers of the events to select your talk, as well as trying to convince attendees your talk is the one they should see.
+Notes
+  Notes will only be seen by reviewers during the CFP process. This is where you should explain things such as technical requirements, why you're the best person to speak on this subject, etc...
+Tags
+  Talk your talk to make it easier for event organizers to be able to find. Examples are "ruby, javascript, rails".
```

Comment:
This is certainly a better list, but I don't think that uncurated tags work very well.

I assume the intention here is to allow attendees to easily see which talks are of interest to them.  If so, if we're going to tag talks, it should be something that we do.  See also my note above about talk titles.